aries-core/src/js/components/external/stories/RangeInput.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Box, Grommet, RangeInput } from 'grommet';
+import { aries } from '../../../../../../aries-site/src/themes/aries';
+
+export default {
+  title: 'RangeInput',
+};
+
+export const Simple = () => {
+  const [value, setValue] = React.useState(5);
+  const onChange = event => setValue(event.target.value);
+
+  return (
+    <Grommet theme={aries} full>
+      <Box align="center" pad="large">
+        <RangeInput value={value} onChange={onChange} />
```

Comment:
what is special about RangeInput that we need to exemplified it so soon? Seems to me all these examples might be too loud.

Reply:
I was just pulling the components that are on the design system site into storybook since it seems like the goal would be to have a closed loop where each component listed on the site would have an associated storybook example.

Would want to discuss with the team if this isn't the direction we want to move.